ui/components/Dataset/index.js

```diff
@@ -19,9 +26,11 @@ function Model({ model }) {
       </Link>
       <p>{description}</p>
       <Flex className="nhsuk-body-s">
-
         <p className={classnames('nhsuk-body-s', styles.noBottom)}>
-          Type: <Tag type={standard_category} classes="nhsuk-body-s">{standard_category}</Tag>
+          Type:{' '}
+          <Tag type={standard_category} classes="nhsuk-body-s">
+            {standard_category}
+          </Tag>
```

Comment:
`{'🤷‍♂️'}`

Reply:
Prettier formats it like this, boo! https://github.com/prettier/prettier/issues/4223 
https://github.com/prettier/prettier/issues/4223#issuecomment-1104943499 *sigh*